bundles/org.openhab.binding.bondhome/src/main/java/org/openhab/binding/bondhome/internal/api/BondSysVersion.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bondhome.internal.api;
+
+import static org.openhab.binding.bondhome.internal.BondHomeBindingConstants.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * This POJO represents the version information of the bond bridge
+ *
+ * The incoming JSON looks like this:
+ *
+ * {"target": "snowbird", "fw_ver": "v2.5.2", "fw_date": "Fri Feb 22 14:13:25
+ * -03 2019", "make": "Olibra LLC", "model": "model", "branding_profile":
+ * "O_SNOWBIRD", "uptime_s": 380, "_": "c342ae74"}
+ *
+ * @author Sara Geleskie Damiano - Initial contribution
+ */
+@NonNullByDefault
+public class BondSysVersion {
+    // The current state hash
+    @SerializedName("_")
+    @Expose(serialize = false, deserialize = true)
+    public String hash = API_HASH;
+
+    @Expose(serialize = true, deserialize = true)
+    public String target = BOND_BRIDGE_TARGET;
+
+    @SerializedName("fw_ver")
+    @Expose(serialize = true, deserialize = true)
+    public String firmwareVersion = BOND_LAST_KNOWN_FIRMWARE;
+
+    @SerializedName("fw_date")
+    @Expose(serialize = true, deserialize = true)
+    public String firmwareDate = BOND_LAST_KNOWN_FIRMWARE_DATE;
```

Comment:
What is the purpose of this default value?

Reply:
Not a clue, but I'll be happy to track it down. Remember that this binding was started by @SRGDamia1, and then picked up by me a long time later after it was apparently abandoned. I suspect that defaults are not necessary.

Reply:
I don't understand the logic, at least. If `fw_ver` and/or `fw_date` would be missing in the JSON, some old hardcoded values would be used. It seems wrong, but maybe there is some reason? I understand that you have resumed the work of @SRGDamia1, and my comments are only towards the code. We should be able to figure out these few remaining open comments, so we can get this merged soon. :)